test/staging/Intl402/Temporal/old/date-time-format.js

```diff
@@ -680,10 +714,11 @@ assert.sameValue(
 );
 
 // should not break legacy Date
-assert.sameValue(us.formatRange(start, end), "12/29/1922 \u2013 12/25/1991");
-assert.sameValue(at.formatRange(start, end), "30.12.1922 \u2013 26.12.1991");
+assert.sameValue(us.formatRange(start, end), `12/29/1922${usDateRangeSeparator}12/25/1991`);
+assert.sameValue(at.formatRange(start, end), `30.12.1922${deDateRangeSeparator}26.12.1991`);
 
-// should throw a TypeError when called with dissimilar types", () => assert.throws(TypeError, () => us.formatRange(Temporal.Instant.from(t1), Temporal.PlainDateTime.from(t2))));
+// should throw a TypeError when called with dissimilar types",
+//  () => assert.throws(TypeError, () => us.formatRange(Temporal.Instant.from(t1), Temporal.PlainDateTime.from(t2))));
```

Comment:
Looks like this is actually tested below.
```suggestion
```

Reply:
Looks like the original port from demitasse tests forgot to remove this commented line. I'll remove it.